src/test/resources/picocompletion-demo_completion.bash

```diff
@@ -256,7 +377,7 @@ function _picocli_picocompletion-demo_sub2_subsub2() {
   local commands=""
   local flag_opts=""
   local arg_opts="-u --timeUnit -t --timeout"
-  local timeUnit_option_args="%2$s" # --timeUnit values
+  local timeUnit_option_args="NANOSECONDS MICROSECONDS MILLISECONDS SECONDS MINUTES HOURS DAYS" # --timeUnit values
```

Comment:
Please leave the `%2$s` variable, it will be replaced by the supported TimeUnit constants for the version of Java that is running the test.